TypeContractor.Tool/Program.cs

```diff
@@ -20,6 +20,7 @@
 var buildZodSchemasOptions = new Option<bool>("--build-zod-schemas", () => false, "Enable experimental support for Zod schemas alongside generated types.");
 var generateApiClientsOptions = new Option<bool>("--generate-api-clients", () => false, "Enable experimental support for auto-generating API clients for each endpoint.");
 var apiClientsTemplateOptions = new Option<string>("--api-client-template", () => "aurelia", "Template to use for API clients. Either 'aurelia', 'react-axios' (built-in) or a path to a Handlebars file, including extension");
+var casingOptions = new Option<string>("--casing", () => "pascal", "Casing to use for generated types. Any of 'pascal' (default), 'camel', 'snake', or 'kebab'");

```

Comment:
Maybe create an enum (like LogLevel or CleanMethod) to get tab completion and help with documentation

Reply:
Yeah, I started with an enum but I couldn't figure out how to add an enum so I can access it inside both the Tool and Lib projects, should I have two enums or should I do some fancy namespace thingy? (I mostly tried to copy what you have in CleanMethod but I don't think that is used in other projects)